examples/dapperish/trivialrecorder.go

```diff
@@ -33,9 +33,9 @@ func (t *TrivialRecorder) SetTag(key string, val interface{}) *TrivialRecorder {
 // RecordSpan complies with the basictracer.Recorder interface.
 func (t *TrivialRecorder) RecordSpan(span basictracer.RawSpan) {
 	fmt.Printf(
-		"RecordSpan: %v[%v, %v us] --> %v logs. std context: %v\n",
+		"RecordSpan: %v[%v, %v us] --> %v logs. std context: %v; baggage: %v\n",
```

Comment:
not needed to fix the build, but wanted to add this since it's good to confirm that baggage propagation works.